tests/python/unit/test_ontology_tagger.py

```diff
@@ -97,4 +94,4 @@ def test_pipeline_components():
     )
     tagger._initialize_tokenizer(SUPPORTED_LANGUAGES_SPACY.keys())
     for language in tagger.tokenizer.spacy_nlp_dict:
-        assert tagger.tokenizer.spacy_nlp_dict[language].pipe_names == ["sentencizer"]
\ No newline at end of file
+        assert tagger.tokenizer.spacy_nlp_dict[language].pipe_names == ["sentencizer"]
```

Comment:
It is not very clear what the test tests based on its name. I would suggest test_initialize_tokenizer.

Reply:
89f22ca

Reply:
super